### ISSUE TITLE
[Accessibility]: Code Insights updating repo filter does not announce changes

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41821]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33499

### Problem description

![PixelSnap 2022-09-20 at 15 44 06](https://user-images.githubusercontent.com/1855233/191360753-a745af7f-0a6a-4b5b-ab88-4d2d92e4f812.png)

Changing either of the repo filters updates the graph preview. But the changes are not announced. A screen reader user would have no idea that these changes did anything.

### Expected behavior

The screen reader announces that the preview has changed. 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
